content/nginxaas-azure/getting-started/ssl-tls-certificates/overview.md

```diff
@@ -242,6 +242,77 @@ The following section describes common errors you might encounter while adding S
    ```
 </details>
 
+2. Integrate with a Private Endpoint to allow NGINXaaS to fetch certificates via Azure Private Link.
+<details>
+<summary>Create a Private Link - Azure CLI</summary>
+
+1. Get the resource ID of the key vault.
+
+   Please ensure the following environment variables are set before copying the below Azure CLI command.
```

Comment:
Our [style guide](https://github.com/nginx/documentation/blob/main/documentation/style-guide.md#:~:text=ensure%20persistent%20connections....%22-,please,-%2C%20thank%20you) suggests that you shouldn't use please (unless the problem is "our fault"). And I think you can be more direct:


```suggestion
   Set the following environment variables before copying the following Azure CLI command.
```